CIP-????/README.md

```diff
@@ -0,0 +1,313 @@
+---
+CIP: XXXX
+Title: Efficient Proofs for Dynamic Sets on Cardano
+Status: Proposed
+Category: Tools
+Authors:
+    - Pal Dorogi <pal.dorogi@gmail.com>
+Implementors: []
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/816
+    - https://github.com/cardano-foundation/CIPs/pull/816
+Created: 2024-05-17
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This Cardano Improvement Proposal (CIP) describes a solution for ensuring the uniqueness (at minting time) of dynamically minted NFTs on the Cardano blockchain. 
+
+The solution uses a new accumulator scheme, leverages a complete binary hash tree for compact accumulation and dynamic updates which relies on a trusted Plutus script, specifically designed to perform accumulator verification.
```

Comment:
Yeah, it's ambiguous and misleading. It simply means, that the deployed Plutus script is secure in the sense that no any adversarial can modify/change it after deployment, but it does not mean that Plutus script itself is secure (it requires some audit etc.) in any way.

Reply:
Ah, is this the script that everyone will use? In that, you aim to create a universal script that others can reference, so that you can onchain proof that projects adhere to this standard.